### DISCUSSION TITLE
Country is in "search" available, but not in "places". Why?

### DISCUSSION THREAD

**Discussion by racecrew** [photoprism/photoprism#3687] [Q&A] [answered]
"Croatia" is in search present, but not in "places". On the picture there is no Croatia in "Länder".

I have done multiple re-index of the pictures, but Croatia doesn't appear. But on the map the pictures from Croatia are shown.

Is it a bug, or am I doing something wrong?

![image](https://github.com/photoprism/photoprism/assets/17234283/ed2371ac-cab5-41fa-8fba-601c89e8341b)

**Comment by lastzero** [comment]
The correct translation is Bundealänder or Regionen. In German, it's still Land Brandenburg. It can be confusing. Don't ask me why.

**Reply by lastzero** [reply]
As a result, pictures without a known state (just a country) won't show up there. We already have a feature request for that.

**Comment by racecrew** [comment]
Ok, if a picture doesn't have any state (Bundesland), and has only a country (Land/Staat), it doesn't appear in the "places"... 

Which PR is that? Thanks!

![image](https://github.com/photoprism/photoprism/assets/17234283/b1dd8e73-1046-4daa-9176-527304008da1)

**Reply by lastzero** [reply, marked as answer]
- https://github.com/photoprism/photoprism/issues/3668

**Comment by racecrew** [comment]
e.g. an iphone picture (.heic) doesn't have any property like state or country, it has only "coordinates". So, photoprism should read out from the coordinates the state and country, right? Or am I missing something here?

**Reply by lastzero** [reply]
Yes. However, you can set the country manually or it can be automatically guessed from a folder name, for example.

**Comment by racecrew** [comment]
Ok, thanks for you answers!
